app/controllers/spree/products_controller_decorator.rb

```diff
@@ -6,12 +6,14 @@ def prepended(klass)
   end
 
   def index
-    load_feed_products if request.format.rss? || request.format.xml?
+    load_feed_products if (request.format.rss? || request.format.xml?)
     respond_to do |format|
       if product_feed
         format.rss { render inline: xml.generate, layout: false }
+        format.xml { render inline: xml.generate, layout: false }
```

Comment:
Do we need the separate `format` calls here? It's been a while since I messed with formats like these, but it seems like this could be just a `render` since it returns the same thing either way.

Reply:
Rails considers `xml` and `rss` different formats, and the only way I've seen to respond to multiple formats is via `format.all`. I guess we could DRY this up to a private method but I didn't figure it was worth the indirection.